content/blog/2021-02-18-disabled-features.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Introducing Disabled Features
+created_at: 2021-02-18
```

Comment:
Two days late?

Reply:
The date is tentative right now, I will change it as soon as v2.25 is released

Reply:
Changed the date to 17th since release is going out today